pages/applications/unassigned.tsx

```diff
@@ -136,7 +119,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 
   const applications =
     reference === '' && status === ''
-      ? await getApplications(page)
+      ? await getApplications(page, 'unassigned')
```

Comment:
That magic string should probably be defined somewhere? An easy, good enough answer might be:
```
function getApplications(page, query: string | "unassigned")
```